pkg/util/helm/helm.go

```diff
@@ -114,6 +115,17 @@ func KlusterToHelmValues(kluster *v1.Kluster, openstack *OpenstackOptions, certi
 				AccessMode: accessMode,
 			},
 			StorageContainer: etcd_util.DefaultStorageContainer(kluster),
+			Openstack: openstackValues{
+				AuthURL:             openstack.AuthURL,
+				Username:            openstack.Username,
+				Password:            openstack.Password,
+				DomainName:          openstack.DomainName,
+				Region:              openstack.Region,
+				ProjectID:           kluster.Spec.Openstack.ProjectID,
+				LbSubnetID:          kluster.Spec.Openstack.LBSubnetID,
```

Comment:
Those three values are not needed, I would remove them

Reply:
@databus23 I removed the unneeded values.